Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -1101,7 +1101,7 @@ void NodeWiseMeshPartitioner::writeElementsASCII(
         for (const auto* node : partition.nodes)
         {
             nodes_local_ids[node->getID()] = node_local_id_offset;
-            node_local_id_offset++;
+            ++node_local_id_offset;
```

Comment:
What's the difference. Add a cppcheck message to the commit message, please. (Btw typo in the commit message)

Reply:
The cppcheck message is:
*Prefix ++/-- operators should be preferred for non-primitive types. Pre-increment/decrement can be more efficient than post-increment/decrement. Post-increment/decrement usually involves keeping a copy of the previous value around and adds a little extra code.*

Typo is fixed.

Reply:
That's the reason, why clang-tidy is better here: `IntegerType` is a `long`. Doesn't matter.